pages/iam/reference-content/understanding-policy-conditions.mdx

```diff
@@ -0,0 +1,134 @@
+---
+meta:
+  title: Understanding policy conditions
+  description: Detailed information on policy conditions within Scaleway IAM.
+content:
+  h1: Understanding policy conditions
+  paragraph: Learn how to use policy conditions to fine-tune access control within Scaleway IAM.
+tags: iam
+dates:
+  validation: 2025-03-03
+categories:
+  - iam
+  - console
+---
+
+A condition is an additional layer of restrictions for your rule. You can configure conditions in IAM policies to allow access to specific user agents, IP addresses and on a given date or time.
+
+At Scaleway, IAM conditions are defined using Common Expression Language (CEL) expressions.
+
+<Message type="tip">
+  Refer to the [How to create a policy](/iam/how-to/create-policy/) and [How to manage policies](/iam/how-to/create-policy/) documentation pages to learn where and how to specify a condition.
+</Message>
+
+## Condition expressions
+
+An expression can be compared to a conditional statement in programming. It is a logical statement that evaluates to either true or false. The result determines whether the permission set defined in the rule is applied or not.
+
+Condition expressions are composed of one or several statements that declare a rule based on attributes. Attributes are like characteristics or properties of a resource or a user. For example, an attribute might be a given date or time, or an IP address.
```

Comment:
Properties of a resource, a user or a request. In fact we are only doing the request part for now :p Resource-based is MIAM and User-based has few use cases